.storybook/config.js

```diff
@@ -1,24 +1,17 @@
 import { configure, addDecorator, addParameters } from '@storybook/react'
 
-import { withInfo } from '@storybook/addon-info'
 import { withA11y } from '@storybook/addon-a11y'
 import { withKnobs } from '@storybook/addon-knobs'
+import { addReadme } from 'storybook-readme'
 
-addDecorator(
-  withInfo({
-    header: true,
-    inline: true,
-    maxPropsIntoLine: 1,
-    maxPropObjectKeys: 10,
-    maxPropArrayLength: 10,
-    maxPropStringLength: 100,
-    propTables: null, // no use for this since we don't use proptypes
-  }),
-)
+addDecorator(addReadme)
 addDecorator(withA11y)
 addDecorator(withKnobs)
 
 addParameters({
+  options: {
+    name: 'BlaBlaCar UI',
```

Comment:
BlaBlaCar UI Library?

Reply:
It would be set in several lines, and it would remove one item visible directly without scrolling